testing/visual-regression/backstop.json

```diff
@@ -714,6 +714,20 @@
       "expect": 0,
       "misMatchThreshold": 0.1,
       "requireSameDimensions": true
+    },
+    {
+      "label": "Components/OISC Warning/With title and description",
+      "url": "http://host.docker.internal:6006/iframe.html?id=components-oisc-warning--with-title-and-description&viewMode=story"
+    },
+    {
+      "label": "Components/OISC Warning/Title only",
+      "url": "http://host.docker.internal:6006/iframe.html?id=components-oisc-warning--title-only&viewMode=story"
+    },
+    {
+      "label": "Components/OISC Warning/Sticky (scrolled)",
+      "url": "http://host.docker.internal:6006/iframe.html?id=components-oisc-warning--sticky&viewMode=story",
+      "selectors": ["viewport"],
```

Comment:
This `["viewport"]` magic selector is needed to test scroll behaviour. Default backstop behaviour is to capture the full element regardless of viewport height.